charts/community-operator/values.yaml

```diff
@@ -62,13 +62,13 @@ database:
 
 agent:
   name: mongodb-agent-ubi
-  version: 107.0.0.8465-1
+  version: 107.0.1.8507-1
 versionUpgradeHook:
   name: mongodb-kubernetes-operator-version-upgrade-post-start-hook
   version: 1.0.8
 readinessProbe:
   name: mongodb-kubernetes-readinessprobe
-  version: 1.0.17
+  version: 1.0.19
```

Comment:
we released 1.0.18 some time ago and just recently we made some changes so I am bumping it to 1.0.19 in the community release as well